controller/article-controller.js

```diff
@@ -11,13 +11,20 @@ function getArticleById(request, response, next){
 }
 
 function getArticle(request, response, next){
-    const { order } = request.query
-    const isOrder = order.toUpperCase()
-    fetchArticle(isOrder).then((articles) => {
-        response.status(200).send({articles})
+    const { topic } = request.query
+    if(Object.keys(request.query).length === 0){
```

Comment:
We probably don't need this conditional. Our model could be invoked with topics every time and behave differently if topics is undefined